.devops/deploy-pipelines.yml

```diff
@@ -0,0 +1,151 @@
+# Azure DevOps pipeline to release a new version and publish the package to the registry
+#
+# The following variables must be configured in the pipeline settings
+#
+# GIT_EMAIL         - when bumping the package version, is the email we author the commit with
+# GIT_USERNAME      - when bumping the package version, is the name we author the commit with
+# GITHUB_CONNECTION - name of the Github service connection used to create a new release; be sure that the related user has appopriate right
+# NPM_CONNECTION    - name of the NPM service connection used to publish the package; be sure that the related user has appopriate right
+# 
+
+variables:
+  NODE_VERSION: '10.14.1'
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+  gitBranchName: ${{ replace(replace(variables['Build.SourceBranch'], 'refs/tags/', ''), 'refs/heads/', '') }}
```

Comment:
Very good! 

Perhaps we can use different naming, as it can be either a branch or a tag. We use `gitReference` for this value in another context, maybe it's fine here as well.

Reply:
Yes, you're right.. I'll fix it!